web-ts/tools/tsconfig.webpack.json

```diff
@@ -1,9 +1,3 @@
 {
-  "compilerOptions": {
-    "module": "commonjs",
-    "target": "es2020",
-    "esModuleInterop": true,
-    "noEmit": true
-  },
-  "files": ["webpack.config.ts"]
+  "extends": "../tsconfig.json"
```

Comment:
I would keep this file standalone. Changes for TS for the app code are not necessarily needed for webpack and vice versa.

We probably need to tune the TS files in the future to add communicability for IE etc.